entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -11,11 +11,11 @@ uid: core/what-is-new/ef-core-3.0/breaking-changes
 > [!IMPORTANT]
 > Please note that the feature sets and schedules of future releases are always subject to change, and although we will try to keep this page up to date, it may not reflect our latest plans at all times.
 
-These are breaks in either API or behavior between the 2.2.x releases and the 3.0.0 release that we believe can affect applications.
+The following API and behavior changes have the potential to break applications developed for EF Core 2.2.x when upgrading them to 3.0.0.
 Changes that we expect to only impact database providers are documented under [provider changes](../../providers/provider-log.md).
-Breaks in new features introduced from one 3.0 preview to another 3.0 preview are not documented here.
+Breaks in new features introduced from one 3.0 preview to another 3.0 preview aren't documented here.
```

Comment:
Not a favor of contractions in this kind of doc. Makes it seem to casual.

Reply:
It is the Microsoft Docs guidelines though.

Reply:
What is the reasoning? Is it a requirement?

Reply:
See https://docs.microsoft.com/style-guide/top-10-tips-style-voice. I was more than once pointed to it by the docs team. I admit I haven't read the whole guide.

Reply:
It seems very geared towards consumer-facing documents and U.I. Personally, I don't think much of it applies to technical docs. To me, being informal and "friendly" to this degree in a technical doc just looks unprofessional.

Reply:
@ajcvickers I don't feel the same way about it, but I am not a native English speaker, so maybe you are right.  

cc @mairaw @Rick-Anderson @tdykstra what is your take from the perspective of the .NET and ASP.NET Core documentation? And your take on Arthur's point?

FWIW, if this is definitely something we shouldn't use, I will create an issue to revert any changes to use contractions in the rest of the EF docs.

Reply:
Contractions are officially required wherever possible in our documentation.  The way the style guide is written implies you aren't even allowed to use your judgment as to when to avoid them.  I tried to get that loosened and they refused. See https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions

Reply:
Yep, @tdykstra is right. Those guidelines are for technical docs too.